File: backjoon/15904/solve.ts
import { readFileSync } from "fs"

const line = readFileSync(0, "utf-8").trim()

const ucpc = line
  .replace(/ /gi, "")
  .replace(/[a-z]/g, "")

if (ucpc == "UCPC") console.log("I love UCPC")
else console.log("I hate UCPC")

File: backjoon/15904/solve.go
package main

import (
	"fmt"
	"regexp"
	"strings"
)

func main() {
	input := ""

	fmt.Scanln(&input)

	reg := regexp.MustCompile("[a-z]")
	shortened := (strings.ReplaceAll(reg.ReplaceAllString(input, ""), " ", ""))


	if (shortened == "UCPC") {
		fmt.Println("I love UCPC")
	} else {
		fmt.Println("I hate UCPC")
	}
}